modules/claims_api/app/sidekiq/claims_api/v2/disability_compensation_pdf_generator.rb

```diff
@@ -0,0 +1,115 @@
+# frozen_string_literal: true
+
+require 'claims_api/v2/disability_compensation_pdf_mapper'
+require 'pdf_generator_service/pdf_client'
+
+module ClaimsApi
+  module V2
+    class DisabilityCompensationPdfGenerator < DisabilityCompensationClaimServiceBase
+      EVSS_DOCUMENT_TYPE = 'L023'
+
+      def perform(claim_id, middle_initial) # rubocop:disable Metrics/MethodLength
+        log_job_progress('526 v2 PDF Generator job',
```

Comment:
I almost wonder if we write some mapping for constants here so they remain consistent and easy to change. or even just one constant here in this file to make it easier.

Rough example: `PDF_GENERATOR = '526 v2 PDF Generator Job'` and pass `PDF_GENERATOR` to each of these.

Not needed, just a thought.

Reply:
that is a good idea